src/main/java/com/codefactorygroup/betting/controller/EventController.java

```diff
@@ -5,6 +5,7 @@
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
+import java.util.Map;
 
 @RestController
```

Comment:
Here you can use  @RequestMapping("/events") in order to avoid duplication at every method lvl